cmd/atlas/internal/cmdapi/migrate_test.go

```diff
@@ -713,6 +713,95 @@ func TestMigrate_ApplyTxModeDirective(t *testing.T) {
 	require.Equal(t, s, `unknown txmode "unknown" found in file directive "20220925094021_second.sql"`)
 }
 
+func TestMigrate_ApplyExecOrder(t *testing.T) {
```

Comment:
Add test that checks the new default is applied.

Reply:
There's one for this (L750~).